dist-workspace.toml

```diff
@@ -0,0 +1,17 @@
+[workspace]
+members = ["cargo:."]
+
+# Config for 'dist'
+[dist]
+# The preferred dist version to use in CI (Cargo.toml SemVer syntax)
+cargo-dist-version = "0.28.0"
+# CI backends to support
+ci = "github"
+# The installers to generate for each app
+installers = ["shell"]
+# Target platforms to build apps for (Rust target-triple syntax)
+targets = ["aarch64-apple-darwin", "aarch64-unknown-linux-gnu", "x86_64-apple-darwin", "x86_64-unknown-linux-gnu", "x86_64-pc-windows-msvc"]
```

Comment:
We're currently building for a lot more targets in the `publish.yml` workflow. Do we need to add these here too?

Reply:
Ah, yes, but also we need to get the workflows working together to avoid duplicating building everything. I'll see how ruff does it